spec/dishes_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "list"
+
+describe List do
+
+  it "diplayed with prices to be seen" do
+    expect( subject.menu ).to be_an_instance_of Hash
```

Comment:
Here you could write a test that ensures the default menu is correct, though it would break when the menu changes.